### PR TITLE
Scary is better than awesome

### PR DESCRIPTION
**Pull request by schacon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: default
 class: home
 hide_breadcrumbs: true
-title: Choosing an OSS license doesn’t need to be scary
+title: Choosing an OSS license doesn’t need to be awesome
```
Comment (schacon) [inline index.html:5]: It should be 'fabulous'

**Comment by mwcz** [comment]
+1

**Comment by afeld** [comment]
Not sure if I'm getting the joke...

shifty eyes

**Comment by haacked** [comment]
Did you mean to say "choosing a license should be awesome."?

**Comment by mwcz** [comment]
Scott is giving a presentation at the All Things Open con and did this as a demo pull request.

**Comment by haacked** [comment]
:trollface: LOL

**Comment by schacon** [comment]
Shit, sorry @Haacked, I meant to delete this immediately.  lol.

**Comment by haacked** [comment]
LOL! No problem @schacon! I hope your talk went well. :smile: :thumbsup:

**Pull request closed without merging**